app/helpers/data_request_helper.rb

```diff
@@ -0,0 +1,6 @@
+module DataRequestHelper
```

Comment:
You could avoid the use of a helper method by creating a new instance variable in the controller method to show whether @recipient_emails is empty before the Branston email gets added in.

You can use helper methods for many things that need to happen in a view, but the best use case is where it is only concerned with how something is displayed and where it needs to be used in several places. The less business logic contained in a helper the better.

Reply:
Either way there should probably be a test for this method, or for the change to the controller if you do that.